File: frmSales2DayCashr.vb
Public Class frmSales2DayCashr

    Private Sub frmSales2DayCashr_FormClosed(sender As Object, e As FormClosedEventArgs) Handles Me.FormClosed
        Me.Dispose()
    End Sub

    Private Sub frmSales2DayCashr_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        DevExpress.XtraEditors.WindowsFormsSettings.DefaultFont = New Font("Courier New", 12)
        GridView1.OptionsView.ShowAutoFilterRow = True
        GridView1.OptionsFind.AlwaysVisible = True
        'TODO: This line of code loads data into the 'DsSale2dayCashr.vwSalesTodayCshr' table. You can move, or remove it, as needed.
        Me.VwSalesTodayCshrTableAdapter.Fill(Me.DsSale2dayCashr.vwSalesTodayCshr)

    End Sub

    Private Sub GridControl1_DoubleClick(sender As Object, e As EventArgs) Handles GridControl1.DoubleClick
        GridControl1.ShowPrintPreview()
    End Sub
End Class

File: DoorsPOS.WinApp/CS/frmSales2DayCashr.cs
// VBConversions Note: VB project level imports
using System.Data;
using System.Collections.Generic;
using PDSA.Data;
using PDSA.WinForms.Forms;
using PDSA.Security;
using System.Diagnostics;
using PDSA.Common;
using PDSA.WinForms;
using PDSA.WinForms.Security;
using PDSA.PDSACryptography;
using DoorsPOS.Utility;
using Microsoft.VisualBasic;
using System.Collections;
using System;
using DoorsPOS.BusinessLayer;
using System.Windows.Forms;
using System.Drawing;
using System.Linq;
using DoorsPOS.DataLayer;
// End of VB project level imports


namespace DoorsPOS.WinApp
{
	public partial class frmSales2DayCashr
	{
		public frmSales2DayCashr()
		{
			InitializeComponent();
		}
		
		public void frmSales2DayCashr_FormClosed(object sender, FormClosedEventArgs e)
		{
			this.Dispose();
		}
		
		public void frmSales2DayCashr_Load(object sender, EventArgs e)
		{
			//TODO: This line of code loads data into the 'DsSale2dayCashr.vwSalesTodayCshr' table. You can move, or remove it, as needed.
			this.VwSalesTodayCshrTableAdapter.Fill(this.DsSale2dayCashr.vwSalesTodayCshr);
			
		}
		
		public void GridControl1_DoubleClick(object sender, EventArgs e)
		{
			GridControl1.ShowPrintPreview();
		}
	}
}
